lib/matplotlib/widgets.py

```diff
@@ -584,6 +587,29 @@ def set_val(self, val):
         if self.eventson:
             self._observers.process('changed', val)
 
+    def update_range(self, valmin, valstep, valmax):
+        """
+        Set the slider range.
+
+        The slider value is coerced to the closest number
+        in the ``valstep`` grid.
```

Comment:
```suggestion
        in the *valstep* grid.
```
Cosmetic, but per our documentation policy we format parameters in italics when in prose.